bot/slash_commands/commands.py

```diff
@@ -9,13 +9,9 @@
 print('commands.py')
 
 SUMMARIZE_SYSTEM_PROMPT = "You are the person responsible for summarizing server messages in a succinct and effective way for the server owner and supporters. Always report the number of messages found."
-SUMMARIZE_SYSTEM_PROMPT = ""
```

Comment:
what is the reason not to provide a system prompt here? i have added it back

Reply:
I guess I was trying to stop it from reporting the number of messages found. It's still doing it, though. I just tried in the general channel and I got:

"Summary of last 10 messages: In the recent channel history, there was a playful exchange about a kazoo tour, .... In total, 7"

Reply:
ok i just noticed the system prompt has this statement "always report the number of messages found." i didnt have that in my local branch when testing. we probably just have to remove that statement.

Reply:
i have tried a lot and im not able to recreate the message count issue locally off the latest master.

i was able to recreate it yesterday before my PR fix.

and per my reply above i cant quite understand from the code why it would still be counting.

just want to make sure if you pushed the latest master to heroku successfully, or if something got tangled up there.

otherwise would need to debug via print statements in production if possible

Reply:
I don't think Taylor has pushed to heroku yet

Reply:
ah ok cool. thought u were saying its still not working after my fix. but understand now what you meant.

it should be good after heroku push.